dr_scaffold/management/commands/dr_scaffold.py

```diff
@@ -51,5 +51,10 @@ def handle(self, *args, **kwargs):
             actions = False
         if kwargs["tests"] == "None":
             tests = False
-        generator = Generator(args[0], args[1], args[2:], actions, tests)
+
+        # Remove : if exists on first and second arguments.
+        first_args = args[0].replace(':', '')
+        second_args = args[1].replace(':', '')
+
+        generator = Generator(first_args, second_args, args[2:], actions, tests)
```

Comment:
I kind of want to keep it like we already made it easier for you to scaffold everything, do your part, and type a correct command bruh 👀  we won't fix your typos too!